app/views/measures/_measure.html.erb

```diff
@@ -50,30 +50,18 @@
 
   <td><%= measure.excluded_country_list %></td>
 
-  <% if TradeTariffFrontend.regulations_enabled? && TradeTariffFrontend.uk_regulations_enabled? %>
-    <td class="numerical">
-      <% if measure.legal_acts.present? && measure.legal_acts.first.has_national_data? %>
```

Comment:
We don't need any of these conditions since uk regulations and eu regulations all have the same api and so the url comes from the same field. This is since @jebw's improvements in using a standard presenter for all legal acts.